src/genpy/rostime.py

```diff
@@ -226,6 +228,9 @@ def to_time(self):
         """
         return self.to_sec()
 
+    def __hash__(self):
+        return super(Time, self).__hash__()
```

Comment:
Sorry, clearly the base class does provide them, because we're calling `super`!

What I meant to say, is why do we need to overload them? This works just fine for me:

```python
#! python3.5

class Base:
    def __hash__(self):
        return 4

class Derived(Base): pass

class DerivedWithOverride(Base):
    def hash(self):
        return super(DerivedWithOverride, self).__hash__()

assert hash(Base()) == hash(DerivedWithOverride()) == hash(Derived())